sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -191,6 +191,7 @@ def __init__(
             network_tracing=network_tracing,
             **kwargs,
         )
+        self._auth_uri = f"sb://{self._address.hostname}{self._address.path}/consumergroups/{self._consumer_group}"
```

Comment:
2 thoughts - could we be resetting the path when we use $default, or do we not need to do this when not using RBAC?

Reply:
oo good thoughts - will check!

Reply:
This should work with the role assigned to just consumer group and when it's assigned at a higher level too. Just needed to make sure that this auth URI wasn't being used for management ops. Seems to be working now.